recordparser.py

```diff
@@ -26,18 +27,19 @@ def skydns_entries(self) -> dict:
             record = self._records[key]
             try:
                 skyrecord = record.skydns()
+                skyrecord['targetstrip'] = 1
```

Comment:
I think this could be better done inside `Record.skydns()`. It is, in the end, part of the record itself (unlike the `managed-by` add-in)

Reply:
`Record.skydns()` returns values, and zones, not etcd keys.

This is a small program but in case it grows and `Record` class is being reused it should be as clean as possible.

Prefix, suffix and suffix trimming od etcd keys are base on record zone and configured in the `RecordParser` class. Choose in which class you want to do the key mangling but I'd prefer where it is now.

Reply:
It is true that the `targetstrip` attribute is coupled to the path where it is stored, which right now it is not managed by `Record`.

Let's leave it as it is, I might refactor that in the future so `Record` is responsible for setting its own path.